core/src/entities/packages/package.rs

```diff
@@ -111,7 +111,7 @@ impl Package {
 
         let mut package = Package {
             id: "".to_string(),
-            kind: PackageKind::Direct,
+            kind: PackageKind::Primary,
```

Comment:
Are we moving from the term "Direct" to "Primary" regarding SBOMs derived from the SBOM providers?

Reply:
I'll admit that it's a bit more meaningful. "Primary", like the main one we are looking at.

Reply:
So, if one has an old run of the Snyk SBOM Provider in their local MongoDB instance, the testing instructions will fail because of this change.  Just a TIL today.

Reply:
Yeah.  We'll need to purge the DB both locally and in dev/prod.  I think Dev/Prod is actually reset right now, so we should be good there.